src/plugins/advanced_settings/public/management_app/lib/to_editable_config.ts

```diff
@@ -41,15 +42,15 @@ export function toEditableConfig({
 
   const validationTyped = def.validation as StringValidationRegexString;
 
-  const conf = {
+  const conf: FieldSetting = {
     name,
     displayName: def.name || name,
     ariaName: def.name || getAriaName(name),
     value,
     category: def.category && def.category.length ? def.category : [DEFAULT_CATEGORY],
     isCustom,
     isOverridden,
-    readonly: !!def.readonly,
+    readOnly: !!def.readonly,
```

Comment:
There was an undetected typo here, as `conf` was not explicitly typed and as `readOnly` is optional.

Note that this didn't have any effect because the fields are already filtered by `!readonly` in an earlier stage anyway.